pkgs/by-name/hp/hpp2plantuml/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "hpp2plantuml";
+  version = "0.8.5";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    hash = "sha256-PfTJmBypI21AAK3sMojygQfrhnRqcMmVCW4dxGfDfQg=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    jinja2
+    cppheaderparser
+  ];
+
+  pythonImportsCheck = [
+    "hpp2plantuml"
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytest
+  ];
+
+  meta = with lib; {
```

Comment:
cold you avoid using `with lib` here because it is a problematic anti-pattern, refer to: https://github.com/NixOS/nixpkgs/issues/208242